frontend/device/kobo/nickel_conf.lua

```diff
@@ -53,9 +53,7 @@ function NickelConf.frontLightLevel.get()
         -- FrontLightState config, so don't normalize the value here yet.
         return tonumber(new_intensity)
     else
-        -- In NickelConfSpec, require("device") won't return KoboDevice
-        local powerd = require("device/kobo/powerd")
-        local fallback_fl_level = powerd.fl_intensity or 1
+        local fallback_fl_level = 20
```

Comment:
I don't know when this is activated exactly, but if this fallback thing is responsible for what I consider the unwanted turning on of my frontlight I'd be a lot unhappier about `20` than about `1`. That is, I don't think I want a fallback but I'd just like it to be off?